app/controllers/api/v1/participant_validation_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ParticipantValidationController < Api::ApiController
+      include ApiTokenAuthenticatable
+
+      def show
+        record = ParticipantValidationService.validate(
+          trn: params[:id],
+          full_name: params[:full_name],
+          date_of_birth: Date.iso8601(params[:date_of_birth]),
```

Comment:
This will blow up if `params[:date_of_birth` is nil.

Reply:
its mandatory on the service, see https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/services/participant_validation_service.rb#L10 so blowing up is expected. the alternative is i can guard and raise relevant 4xx errors back to the client?